cpp/arcticdb/CMakeLists.txt

```diff
@@ -160,11 +160,19 @@ if (NOT MSVC)
     add_compile_options("-fno-omit-frame-pointer")
 else()
     add_compile_options(
-        /WX
         /wd4250 # Suppress inherits via dominance warning
-        /we4189 # Unused local variable is an error
         /we4100 # Unused function parameter is an error
+        # For now, we do not want to treat warnings as errors for those builds.
```

Comment:
Why? What is the difference between vcpkg and conda in this regard?

Reply:
This must only be set for the conda-forge build, and we need to understand the origin of warning for them.